querymonitor/README.md

```diff
@@ -2,7 +2,8 @@
 
 ## Synopsis
 Runs (as a daemon) and periodically collects live query info into csv (.gz) files.
-Personably Identifyable information (PII) is removed from each query by truncating the WHERE clause.
+Personably Identifyable information (PII) can be removed from each query by truncating the WHERE clause.
+Built-in --ANALYSIS mode.

```

Comment:
The option says "--ANALYSIS" here but line 62 says "--ANALYZE"

Reply:
Fixed in next check-in